docs/develop/index.md

```diff
@@ -10,49 +11,67 @@ import { SdkExampleTabs } from "@site/src/components/SdkExampleTabs";
 
 <SdkExampleTabs snippetId={'API_CredentialProviderFromEnvVar'} />
 
-If you're storing your Momento auth token in a secret manager such as AWS Secret Manager or a config file, you will first retrieve the credentials from there and then instantiate a `CredentialProvider` from a string:
+If you're storing your Momento auth token in a secret manager such as [AWS Secret Manager](https://aws.amazon.com/secrets-manager/), [GCP Secret Manager](https://cloud.google.com/secret-manager), or a local config file, you must first retrieve the credentials from there and then instantiate a `CredentialProvider` from a string, like this:
 
 <SdkExampleTabs snippetId={'API_CredentialProviderFromString'} />
 
-For an example of how to retrieve credentials from AWS Secrets Manager, see our page on [Retrieving a Momento auth token from AWS Secrets Manager](/develop/integrations/aws-secrets-manager).
+For an example of how to retrieve credentials from AWS Secrets Manager, see [Retrieving a Momento auth token from AWS Secrets Manager](/develop/integrations/aws-secrets-manager).
 
-For general information on creating and refreshing Momento auth tokens, see our page on [Momento authentication with expiring tokens](/develop/guides/working-with-momento-auth-tokens).
+For general information on creating and refreshing Momento auth tokens, see [Momento authentication with expiring tokens](/develop/guides/working-with-momento-auth-tokens).
 
 ## Client configuration objects
 
-You will also need to create a `Configuration` object to configure your Momento Cache client. `Configuration` objects control things such as timeouts, retries, logging, middleware, and more.
+`Configuration` objects contain the settings necessary for how a Momento client should connect to Momento services. The `Configuration` object controls settings such as timeouts, retries, logging, middleware, and more.
+
+Each SDK contains pre-built `Configuration` objects to help get you up and running as quickly as possible. We did the hard work of tuning for various deployment environments so you can focus on the things unique to your business. (We even have a blog series about it! [Shockingly simple: Cache clients that do the hard work for you](https://www.gomomento.com/blog/shockingly-simple-cache-clients-that-do-the-hard-work-for-you))
+
+Pre-built configurations come with a `latest()` version, which will always give you the latest recommended configuration for a given environment. For example,
+
+`configuration: Configurations.InRegion.Default.latest(),`
```

Comment:
If you use the SdkExampleTabs widget for this then we will have the syntax available in the other languages besides just JS, but, up to you

Reply:
Do we have to add something to the SDK repos to show this line?

Reply:
I have the v1s already:

https://github.com/momentohq/client-sdk-javascript/blob/main/examples/nodejs/doc-examples-js-apis.ts#L45-L55

but yeah you would need to add one for `latest`.

i'm writing docs for this code injection stuff now.

Reply:
ok, this is fixed.